src/plots/cartesian/layout_attributes.js

```diff
@@ -698,6 +698,15 @@ module.exports = {
             'In other cases the default is *hide past div*.'
         ].join(' ')
     },
+    drawminorticklabel: {
+        valType: 'integer',
+        editType: 'calc',
+        description: [
+            'Instead of drawing the major tick label, draw the label for the minor tick',
+            'that is n positions away from the major tick. E.g. to always draw the label for the',
+            'minor tick before each major tick, choose `drawminorticklabel` -1'
+        ].join(' ')
+    },
```

Comment:
Does "has no effect for category/multicategory/log axes" still apply?
Would adding that here be clear?

"Has no effect on "multicategory", "category", or "log" axes."

Reply:
In fact, after discussion with @alexcjohnson it's only for period axes. I updated the text accordingly.